codecoverage.py

```diff
@@ -118,9 +119,12 @@ def generate_info(grcov_path):
             i += 1
             time.sleep(60)
         mod_env['PATH'] = one_click_loaner_gcc + ':' + mod_env['PATH']
-
-    fout = open("output.info", 'w')
-    cmd = [grcov_path, '-t', 'lcov', '-p', '/home/worker/workspace/build/src/']
+    fout = open("output.{}".format(output_format), 'w')
+    if output_format == 'info':
```

Comment:
Instead of doing this, let's do:
```
cmd = [grcov_path, '-t', output_format, '-p', '/home/worker/workspace/build/src/']
if 'coveralls' in output_format:
    cmd += ['--token', 'UNUSED', '--commit-sha', 'UNUSED']
```

So we let the caller decide what format they want (either `lcov`, or `coveralls`, or anything else supported by grcov).